mono/mini/method-to-ir.c

```diff
@@ -8938,6 +8938,7 @@ mono_method_to_ir (MonoCompile *cfg, MonoMethod *method, MonoBasicBlock *start_b
 				}
 			}
 
+#ifdef ENABLE_NETCORE
```

Comment:
_why_ do we keep disabling these optimizations for desktop?

Reply:
cc @slluis

Reply:
cc @marek-safar

Reply:
This one does not need to be ENABLE_NETCORE

Reply:
There are quite a few other PRs than went in over the past months which have been enabled in .netcore only and I think they shouldn't be.

I can compile a list, they would be nice to have for vsmac too

Reply:
Ah ok, will enable it back. I thought we only do such changes for netcore only.

Reply:
Having them ifdef NETCORE reduces the risk of regression in the production version.